### PR TITLE
Question Service Implementation

### PR DESCRIPTION
**Pull request by dedsecrattle**
- Updated Question schema and logic
- Added .env template

### CODE REVIEW

File: backend/question-service/.env_template
Comment (nknguyenhc) [inline backend/question-service/.env_template:1]: Better re-named to `.env.sample` or `example.env`

File: backend/question-service/package.json [hunk 1]
```diff
@@ -4,20 +4,23 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon ./src/server.ts"
```
Comment (nknguyenhc) [inline backend/question-service/package.json:8]: Better have a `start` script for deployment as well.

File: backend/question-service/.env_template
Comment (dedsecrattle) [reply backend/question-service/.env_template:1]: sure

File: backend/question-service/package.json (hunk 1 above)
Comment (dedsecrattle) [reply backend/question-service/package.json:8]: yeah , will push will the Dockerisation

**Review by Singa-pirate — APPROVED** [review]
LGTM!

**Pull request merged**